lib/cogs/Lobbies.py

```diff
@@ -350,6 +350,19 @@ def __init__(self, LobbyPrompt) -> None:
     async def on_submit(self, interaction: discord.Interaction):
         await interaction.response.defer(thinking=False)
         self.LobbyPrompt.clear_items()
+        lobby_details = LobbyPrompt.get_lobby_details(interaction.user)
+
+        if lobby_details != {}:
+            await self.LobbyPrompt.delete_prompt()
+            return await interaction.followup.send(

```

Comment:
does `interaction.followup()` work despite the prompt and presumably the interaction being deleted/removed?

Reply:
Yep this is fine